src/settings/tesseractparameterstablewidget.h

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright © 2018-2021 Hennadii Chernyshchyk <genaloner@gmail.com>
+ *
+ *  This file is part of Crow Translate.
+ *
+ *  Crow Translate is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Crow Translate is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a get of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef TESSERACTPARAMETERSTABLEWIDGT_H
+#define TESSERACTPARAMETERSTABLEWIDGT_H
+
+#include <QTableWidget>
+
+class TesseractParametersTableWidget : public QTableWidget
+{
+public:
+    TesseractParametersTableWidget(QWidget *parent = nullptr);
+
+    void addParameter(const QString &key, const QVariant &value);
+    void addParameter();
```

Comment:
Let's just turn these functions into one:
```suggestion
    void addParameter(const QString &key = {}, const QVariant &value = {});
```